OpenSim/Simulation/Model/ControllerSet.h

```diff
@@ -83,6 +84,7 @@ OpenSim_DECLARE_CONCRETE_OBJECT(ControllerSet, ModelComponentSet<Controller>);
     void constructStorage();
     void storeControls( const SimTK::State& s, int step );
     void printControlStorage( const std::string& fileName) const;
+    TimeSeriesTable getControlTable() const;
```

Comment:
For consistency with the methods in this class and in Model, this should be `getControlsTable()`.